examples/src/pages/MapWithDrawTool.tsx

```diff
@@ -118,6 +118,33 @@ const MapWithDrawTool: React.FC = () => {
     }
   }, [mapInstance])
 
+  // Maybe move this to a different example
+  // The same result can be achieved for a Polyline
+  const setInitialDrawing = () => {
+    const polybounds = [
+      { lat: 52.375120046525105, lng: 4.889097815688928 },
+      { lat: 52.370002964209455, lng: 4.885899048674768 },
+      { lat: 52.371241385470576, lng: 4.900439799661425 },
+      { lat: 52.3761937456813, lng: 4.9003854485337985 },
+    ]
+
+    function createPolygon(
+      coordinates: Array<{ lat: number; lng: number }>,
+    ): PolygonType {
+      // @ts-ignore
+      const bounds = coordinates.map(({ lat, lng }) => [lat, lng]) as LatLng[]
```

Comment:
I think for the example, these functions are not needed, just simply create a static polygon like this:
```suggestion
      const existingPolygon = L.polygon([[52.375120046525105, 4.889097815688928], [52.370002964209455, 4.885899048674768], [52.371241385470576, 4.900439799661425], [52.3761937456813, 4.9003854485337985]], {
        color: themeColor('support', 'invalid')({ theme: ascDefaultTheme }),
        bubblingMouseEvents: false,
      }) as PolygonType
```